test/unit/ReportTest.php

```diff
@@ -16,142 +15,178 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
  * Copyright (c) 2013 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
- *
  */
 
+declare(strict_types = 1);
+
 use oat\generis\test\TestCase;
-use common_report_Report as Report;
+use oat\oatbox\reporting\Report;
 
 class ReportTest extends TestCase
 {
-    
-    public function testBasicReport()
+    public function testBasicReport(): void
     {
-        $report = new common_report_Report(common_report_Report::TYPE_SUCCESS, 'test message');
-        $this->assertFalse($report->hasChildren());
-        $this->assertEquals('test message', (string)$report);
-        $this->assertEquals('test message', $report->getMessage());
-        $this->assertEquals(common_report_Report::TYPE_SUCCESS, $report->getType());
+        $report = new Report(Report::TYPE_SUCCESS, 'test message');
+
+        self::assertFalse($report->hasChildren());
```

Comment:
why use `self` instead of `$this`?

Reply:
PHPUnit provides such methods as static, phpstorm in inspections says that more affective call it as static. I tend to agree with it.